Makefile

```diff
@@ -61,65 +63,70 @@ BUILD_IMAGE ?= golang:1.9-alpine
 # If you want to build AND push all containers, see the 'all-push' rule.
 all: build
 
-build-%:
-	@$(MAKE) --no-print-directory ARCH=$* build
+$(addprefix build-, $(ALL_PLATFORMS)):
+	@$(MAKE) --no-print-directory ARCH=$(word 3,$(subst -, ,$@)) OS=$(word 2,$(subst -, ,$@)) build
 
 container-%:
```

Comment:
do we need the same transform for these rules?

Reply:
Nope, leftover from prior iteration. Reverted to simpler version.